adaptivefiltering/segmentation.py

```diff
@@ -100,12 +112,42 @@ def __geo_interface__(self):
         }
 
 
-class InteractiveMap:
-    def __init__(self, dataset=None, segmentation=None):
+def get_min_max_values(segmentation):
+    # goes over all features in the segmentation and return the min and max coordinates in a dict.
+    min_max_dict = {"minX": [], "maxX": [], "minY": [], "maxY": []}
+    for feature in segmentation["features"]:
+        coord_array = np.asarray(feature["geometry"]["coordinates"])
+        min_max_dict["minX"].append(np.min(coord_array, axis=1)[0, 0])
+        min_max_dict["minY"].append(np.min(coord_array, axis=1)[0, 1])
+        min_max_dict["maxX"].append(np.max(coord_array, axis=1)[0, 0])
+        min_max_dict["maxY"].append(np.max(coord_array, axis=1)[0, 1])
+    for key, value in min_max_dict.items():
+        if "min" in key:
+            min_max_dict[key] = min(value)
+        elif "max" in key:
+            min_max_dict[key] = max(value)
+
+    return min_max_dict
+
+
+def swap_coordinates(segmentation):
+    new_features = copy.deepcopy(segmentation["features"])
+
+    for feature, new_feature in zip(segmentation["features"], new_features):
+        coord_array = np.asarray(feature["geometry"]["coordinates"])
+        coord_array[:, :, [0, 1]] = coord_array[:, :, [1, 0]]
+        new_feature["geometry"]["coordinates"] = coord_array.tolist()
+    return Segmentation(new_features)
+
+
+class Map:
+    def __init__(self, dataset=None, segmentation=None, in_srs=None):
         """This class manages the interactive map on which one can choose the segmentation.
             It can be initilized with a dataset from which it will detect the boundaries and show them on the map.
             Or it can be initilized with a segmentation which will also be visualized on the map.
             There can be multiple polygons in the Segmentation and all will drawn. The
```

Comment:
`The` dangles here and the added lines are not full sentences (should end with `.`).